app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -35,6 +35,8 @@ class AdministratorControlsViewModel @Inject constructor(
   private val loadProfileListListener = activity as LoadProfileListListener
   private val showLogoutDialogListener = activity as ShowLogoutDialogListener
   private lateinit var userProfileId: ProfileId
+
+  /** Sets the Index for the currently selected fragment. */
```

Comment:
``index`` not ``Index``. Ditto elsewhere (please only capitalize types or proper nouns).

Reply:
done